src/FffGcodeWriter.cpp

```diff
@@ -785,6 +785,7 @@ void FffGcodeWriter::processSkin(GCodePlanner& gcode_layer, SliceMeshStorage* me
         if (bridge > -1)
         {
             pattern = EFillMethod::LINES;
+            skin_angle = bridge;
```

Comment:
Seeing this commit I conclude that bridging simply didn't work at all.
Thanks for this fix!